docs/integrations/ai-engines/timegpt.mdx

```diff
@@ -112,9 +112,8 @@ CREATE VIEW btcusdt_recent AS (
 This view is going to be joined with the model to get forecasts:
 
 ```sql
-SELECT to_timestamp(cast(m.open_time as bigint)) AS open_time,
-       m.open_price,
-       m.open_price_explain
+SELECT m.open_time ,
```

Comment:
Let's make this change in the README of TimeGPT as well.

Reply:
The README doesn't follow up the same structure as docs. Maybe we change the README and make it same as docs?

Reply:
Yes - let's do that.